package.json

```diff
@@ -49,6 +49,16 @@
                     "type": "string",
                     "default": "",
                     "description": "The name of the Bazel executable. This may be an absolute path, or a simple name that will be searched for on the system path. If empty, \"bazel\" on the system path will be used."
+                },
+                "bazel.buildifierExecutable": {
+                    "type": "string",
+                    "default": "",
+                    "markdownDescription": "The name of the Buildifier executable. This may be an absolute path, or a simple name that will be searched for on the system path. If empty, \"buildifier\" on the system path will be used.\n\nBuildifier can be downloaded from https://github.com/bazelbuild/buildtools/releases."
```

Comment:
does their flavor of markdown auto link this?

Reply:
Yes!